.github/workflows/release.yml

```diff
@@ -199,7 +199,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
I'd be in favour of also dropping the Python 3.6 wheels for macOS and Windows and updating the the required minimum Python version:
https://github.com/larq/compute-engine/blob/3826efcc984136e3f7814f1ab65fd6d4b4636389/setup.py#L33

Python 3.6 has reached its end of life in the end of Dec 2021, so I think it makes sense to completely drop it. @larq/compute-engine What do you think?

Reply:
Done, 3.6 is now no longer supported.